cmd/kube-apiserver/app/patch_openshift.go

```diff
@@ -5,29 +5,14 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	clientgoinformers "k8s.io/client-go/informers"
 	"k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver"
-	"k8s.io/kubernetes/pkg/master"
 )
 
 var OpenShiftKubeAPIServerConfigPatch openshiftkubeapiserver.KubeAPIServerConfigFunc = nil
 
-type KubeAPIServerServerFunc func(server *master.Master) error
-
-func PatchKubeAPIServerConfig(config *genericapiserver.Config, versionedInformers clientgoinformers.SharedInformerFactory, pluginInitializers *[]admission.PluginInitializer) (genericapiserver.DelegationTarget, error) {
+func PatchKubeAPIServerConfig(config *genericapiserver.Config, versionedInformers clientgoinformers.SharedInformerFactory, pluginInitializers *[]admission.PluginInitializer) error {
```

Comment:
@marun we should squash this with the commit introducing this.

Reply:
I can't see the commit you're referring to. Are you asking that I squash the following?

- UPSTREAM: <carry>: openshift-kube-apiserver: add kube-apiserver patches
- UPSTREAM: <carry>: kube-apiserver: allow injection of kube-apiserver options